examples/contentful-simple/README.md

```diff
@@ -0,0 +1,247 @@
+│   │   ├── [slug]/               # Dynamic blog post pages
+│   │   └── page.tsx              # Blog index page
+│   ├── api/makeswift/            # Makeswift API routes
+│   ├── [[...path]]/              # Dynamic Makeswift routes
+│   ├── layout.tsx                # Root layout
+│   └── globals.css               # Global styles
+├── components/                   # React components
+│   ├── BlogContentWithSlot/      # Main blog component with slot
+│   ├── Section/                  # Section components
+│   └── Warning/                  # Error components
+├── lib/                          # Utility functions and configs
+│   ├── contentful/               # Contentful utilities
+│   │   ├── queries/              # GraphQL queries
+│   │   ├── client.ts             # GraphQL client
+│   │   ├── fetchers.ts           # Data fetching functions
+│   │   ├── provider.tsx          # React context provider
+│   │   └── utils.ts              # Formatting utilities
+│   └── makeswift/                # Makeswift configuration
+├── generated/                    # Auto-generated TypeScript types
+│   └── contentful.ts             # Contentful GraphQL types
+├── vibes/soul/                   # Pre-built UI components
+│   ├── primitives/               # Basic UI primitives
+│   └── sections/                 # Layout sections
+└── env.ts                        # Environment variable validation
+```
+
+## Quick Start
+
+### 1. Clone the repository
+
+```bash
+   npx makeswift@latest init --example=contentful
+```
+
+### 2. Install dependencies
+
+```bash
+pnpm install
+```
+
+### 3. Configure environment variables
+
+Create a `.env.local` file and add your credentials:
+
+```
+MAKESWIFT_SITE_API_KEY=your_makeswift_api_key
+CONTENTFUL_SPACE_ID=your_space_id
+CONTENTFUL_ACCESS_TOKEN=your_access_token
+```
+
+### 4. Set up your content model in Contentful
+
+Create the following content types in your Contentful space:
+
+**Author Content Type:**
+
+- **Name\*** (Short text) - The author's name (mark as "Entry title")
+- **Slug\*** (Short text) - URL-friendly identifier
+- **Job Title** (Short text) - The author's job title
+- **Description** (Rich text) - Author bio
+- **Avatar** (Media) - Profile image
+
+**BlogPost Content Type:**
+
+- **Title\*** (Short text) - Post title (mark as "Entry title")
+- **Description** (Long text) - Brief summary
+- **Slug\*** (Short text) - URL-friendly identifier
+- **Feed Date\*** (Date & time) - Publication date
+- **Body** (Rich text) - Main content
+- **Author** (Reference) - Reference to Author content type
+- **Banner** (Media) - Hero image
+
+Add sample content for both Authors and BlogPosts to test the integration.
+
+### 5. Generate TypeScript types
+
+```bash
+pnpm run codegen-ts
+```
+
+This command watches for changes and regenerates types automatically.
+
+### 6. Run the development server
+
+```bash
+pnpm dev
+```
+
+## Building Blog Pages in Makeswift
+
+### Blog Index Page (`/blog`)
```

Comment:
We'll have to update the text below to reflect the code changes in the other comments.